src/OrchardCore.Modules/OrchardCore.Liquid/Startup.cs

```diff
@@ -106,3 +113,32 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddLiquidFilter<ShortcodeFilter>("shortcode");
     }
 }
+
+[RequireFeatures("OrchardCore.Resources")]
+public sealed class ResourcesStartup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddLiquidFilter<AppendVersionFilter>("append_version")
+            .AddLiquidFilter<ResourceUrlFilter>("resource_url");
```

Comment:
Don't chain

Reply:
I don't think it's an issue, second I just copied from the previous code